frontend/src/libs/components/button/styles.module.scss

```
@@ -0,0 +1,14 @@
+.button {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19.2px;
```

let's use integer values
```suggestion
  line-height: 19px;
```